visualization/include/pcl/visualization/interactor_style.h

```diff
@@ -250,6 +260,11 @@ namespace pcl
           modifier_ = modifier;
         }
 
+        /** \brief ID used to fetch/display the look up table on the visualizer
+         * It should be set by PCLVisualizer \ref setLookUpTableID
+         * @note If empty, a random actor added to the interactor will be used */
+        std::string lut_actor_id_;
```

Comment:
I thought that a getter/setter would be add useless complexity so I made it public.
The PCLVisualizer directly modifies the string [here](https://github.com/PointCloudLibrary/pcl/pull/1241/files#diff-9c03ff6eb25cf5a57bad531b4ab8d9f3R4263) for example.